databricks/koalas/testing/doctest_main.py

```diff
@@ -0,0 +1,6 @@
+
+if __name__ == "__main__":
+    import doctest
+    import databricks.koalas as ks
+    from databricks.koalas import frame, series
+    doctest.testmod(frame, extraglobs={"ks": ks})
```

Comment:
This line should be repeated for every module that needs testing. One cannot rely on nosetest to automatically discover all the modules. On the bright side, there is no need to change any of the other files.